### DISCUSSION TITLE
Best way to reference to a GitHub file from another account?

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#28] [Q&A]
Hello @eculler and @nquarder and colleagues, I've created an entry in the Discussion / Show and Tell in order to show that I've made a guide to build our GitHub Pages, but instead of putting all the text in the editing window of this repo, I've put a link to the Tutorial.md file from my personal repo, because it'll probably be change to complete or fix it and I plan to change it and make it be updated automatically wherever there is a link to it. The issue is that I could not find the correct link. The "Copy Path" option gets a relative path and does not work. The "Copy permalink" seems to work but when I change the file the link is not equal to the one I got in the first time, so I have to change the referenced link anyway. Any idea on how I can reference a file in another account in a way that it is always up to date?

**Comment by oye-bobs** [comment]
I'm also curious to know about this.

**Comment by eculler** [comment]
@Garcia-INPE I'm not quite sure what you mean here. Do you want to link to a rendered page from a different repository (e.g. your-username.github.io/repo_name/file_name.html)? Or are you trying to link to the file in the repository itself?
